src/components/Loader/peopleTable.tsx

```diff
@@ -0,0 +1,236 @@
+import { useEffect, useState } from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from './Loader';
+
+export const PeopleTable: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const { personId } = useParams();
+  const selectedPerson = people.find(person => person.slug === personId);
+  const [isPeopleLoaded, setIsPeopleLoaded] = useState(true);
+  const [isPeopleLoading, setIsPeopleLoading] = useState(false);
+
+  const loadPeople = async () => {
+    setIsPeopleLoading(true);
```

Comment:
The top of the table appears for a moment before loading. Better make `isPeopleLoading` true by default and set `false` only in `finally`. 
![image](https://github.com/mate-academy/react_people-table-basics/assets/143965870/b5f343dd-4c2b-4da1-9417-d608f3e20af8)